File: LUYENCODE/PROBLEMS/DK10 - Tìm số ngày của tháng/DK10.py
m, y = map(int, input().split())
if(m >= 1 and m <= 12 and y > 0):
    if(((y % 400 == 0) or (y % 4 == 0)) and (y % 100 != 0) and m == 2):
        print("29",end="")
    elif(m == 2):
        print("28", end="")
    elif((m == 1 or m == 3 or m == 5 or m == 7 or m == 8 or m == 10 or m == 12)):
        print("31", end="")
    elif((m == 4 or m == 6 or m == 9 or m == 11)):
        print("30", end="")
    else:
        print("INVALID", end="")
else:
    print("INVALID", end="")

File: LUYENCODE/PROBLEMS/DK10 - Tìm số ngày của tháng/DK10.cs
using System;
namespace Ngaythangnam
{
    public class Program
    {
        static void Main(string[]args)
        {
            string[] input = Console.ReadLine().Split(' ');
            int m = int.Parse(input[0]);
            int y = int.Parse(input[1]);
            if(m >= 1 && m <= 12 && y > 0)
            {
                if(((y % 400 == 0) || (y % 4 == 0)) && (y % 100 != 0) && m == 2)
                    Console.Write("29");
                else if(m == 2)
                    Console.Write("28");
                else if((m == 1 || m == 3 || m == 5 || m == 7 || m == 8 || m == 10 || m == 12))
                    Console.Write("31");
                else if((m == 4 || m == 6 || m == 9 || m == 11))
                    Console.Write("30");
                else 
                    Console.Write("INVALID");
            }
            else
                Console.Write("INVALID");
        }
    }
}